bsp4j/src/main/java/ch/epfl/scala/bsp4j/CppBuildServer.java

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.scala.bsp4j;
+
+import org.eclipse.lsp4j.jsonrpc.services.JsonRequest;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface CppBuildServer {
+
+    @JsonRequest("buildTarget/cppDependenciesSources")
+    CompletableFuture<CppDependenciesSourcesResult> cppDependenciesSources(CppDependenciesSourcesParams params);
```

Comment:
I think there is probably a better name for this but I can't find them.
The goal of this is to provide a means of setting the initial path an include should have. In c++ build systems, such as bazel it is to common for the import path to depend on the name of the path since c++ relies on paths for imports. An import in bazel, for instance, requires the following:
```c++
#include "<package_name>/path_to_header"
```